code-challenges/401/arrayReverse/array-reverse.js

```diff
@@ -0,0 +1,18 @@
+// Write a function that takes an array as an argument. Without utilizing any of the methods in my language, return an array with elements in reverse order.
+
+const arr = [1, 2, 3, 4, 5, 6];
+
+const initial = 0;
+const lastElement = arr.length -1;
+
```

Comment:
While const declaration is fine for array object, it will not work for primitive types like integers since the integer 'initial' and 'lastElement' needs to be re-assigned to increment/decrement inside of the while loop.

Reply:
Oh! That is good to know! I will not apply const to primitive types in the future.